style.css

```diff
@@ -2609,6 +2608,21 @@ article.panel-placeholder {
 		font-size: 16px;
 		font-size: 1.0rem;
 	}
+
+	/* Gallery Columns */
+	
+	.gallery-item {
+		max-width: 25%;
+	}
+
+	.gallery-columns-3 .gallery-item {
+		max-width: 33.33%;
+	}
+
+	.gallery-columns-4 .gallery-item {
+		max-width: 25%;
+	}
+
```

Comment:
Small nitpick: this space can probably go, too - all other `@media` queries finish with the last two `}` on different lines, but without an extra line break between them.